.xo-config.js

```diff
@@ -22,7 +18,6 @@ module.exports = {
 				namedComponents: 'function-declaration'
 			}
 		],
-		'react/jsx-first-prop-new-line': 'error',
```

Comment:
Did you disable this because you enabled it in xo?

Reply:
Yup: https://github.com/xojs/eslint-config-xo-react/blob/93f78865088aa5deab246e1cc4f40959fc5ff5be/index.js#L101